README.md

```diff
@@ -71,29 +75,6 @@ carthage update
 ```
 In your application targets “General” tab under the “Linked Frameworks and Libraries” section, drag and drop lottie-ios.framework from the Carthage/Build/iOS directory that `carthage update` produced.
 
-### Binary dependency (`xcframework`)
```

Comment:
I don't think we need to recommend this approach anymore, since it was primarily a workaround for the large repo size when using SPM. I think the UX for integrating with the lottie-spm repo is much better than manually managing XCFrameworks / checksums. Using lottie-spm also probably makes it easier for folks to stay up-to-date (since the standard `swift package update` flow still works).